File: solutions/csharp/raindrops/1/Raindrops.cs
using System.Collections.Generic;

public static class Raindrops
{
    private static readonly Dictionary<int, string> Sounds
        = new ()
        {
            { 3, "Pling" },
            { 5, "Plang" },
            { 7, "Plong" }
        };

    public static string Convert(int number)
    {
        var sound = "";

        foreach (var key in Sounds.Keys)
        {
            if (number % key == 0)
                sound += Sounds[key];
        }

        return string.IsNullOrEmpty(sound)
            ? number.ToString()
            : sound;
    }
}

File: solutions/javascript/raindrops/1/raindrops.js
const sounds = {
  3: 'Pling',
  5: 'Plang',
  7: 'Plong'
};

export const convert = (number) => {
  let sound = '';

  for (const key in sounds) {
    if (number % key === 0) {
      sound += sounds[key];
    }
  }

  return sound ? sound : number.toString();
};
